.travis.yml

```diff
@@ -1,6 +1,22 @@
-language: node_js
-node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-  - "4.2.4"
+language: c++
+env:
+  - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
+  - NODE_VERSION="0.12"
+  - NODE_VERSION="4.2.4"
```

Comment:
I think it's at 4.2.4 because that's the latest LTS (@arturadib set that up). I'll add a more recent version as well though.